src/plays/personal-profile-card/PersonalProfileCard.tsx

```diff
@@ -0,0 +1,66 @@
+import PlayHeader from "common/playlists/PlayHeader";
+import React, { useState, ChangeEvent, useEffect } from 'react'
+
+import * as Storage from "./local-storage"
+import ProfileType from './types'
+import ProfileCard from './components/profile-card'
+import ProfileForm from './components/profile-form'
+import './index.css'
+
+const PersonalProfileCard = (props: any) => {
+  const [profileCard, setProfileCard] = useState<ProfileType>(null)
```

Comment:
type of the state is `ProfileType` but why initial value is null? it should expect default state object with keys only (defined on ProfileType)

Reply:
Define  default state.